src/main/steps/applicant/contact-preferences/contact-post-success/content.test.ts

```diff
@@ -16,10 +16,10 @@ const en = {
 };
 
 const cy = {
-  caption: `Case number - welsh #${caseNumber}`,
-  title: 'Contact preferences updated -welsh',
-  text: 'You will receive updates on the case by post. - welsh',
-  continue: 'Continue - welsh',
+  caption: `Rhif yr achos #${caseNumber}`,
+  title: 'Dewisiadau cyswllt wedi’u diweddaru',
+  text: 'Byddwch yn cael diweddariadau am yr achos drwy’r post.',
```

Comment:
Does it require post at the end?

Reply:
Yes, it's in the translation given in the excel document